### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by aliok**
Some spelling and formatting fixes.
Also added a little bit of more documentation about 1-2 things.

### CODE REVIEW

**Comment by aliok** [comment]
ping @wtrocki @luigizuccarelli

**Comment by aliok** [comment]
This one will need a rebase once https://github.com/aerogear/digger-jenkins/pull/5 is merged.
Wait until that one is merged before reviewing!

File: docker/README.md [hunk 1]
```diff
@@ -1,54 +1,78 @@
...
-Install s2i build tool from: https://github.com/openshift/source-to-image/releases/tag/v1.1.3 
-Execute build command:
+
+## AeroGear Digger Jenkins Master OpenShift Docker image
+
+This is the source for image in Docker Hub: <https://hub.docker.com/r/aerogear/jenkins-1-centos7/>
+
+Unless you need to change the Jenkins Docker image, you don't need to do any of the things required here.
+
+From a user's point of view, you don't need to do anything here in order to build apps using AeroGear Digger.
```
Comment (wtrocki) [inline docker/README.md]: Users would not see this. This is for devs I think. Writing here some enduser guide is not good idea. First of all we need to document this for ourselves.

File: docker/README.md [hunk 2]
```diff
@@ -1,54 +1,78 @@
...
+3. Extracting Jenkins configuration
 
+```
     oc project {jenkins project}
     oc rsh {jenkins-pod}
     cat /var/lib/jenkins/configuration.xml
+```
 
-Copy configuration into the repository and build new image.
+Then, copy configuration into the repository and build new image.
```
Comment (wtrocki) [inline docker/README.md]: I forgot to say where. Would be got to try this and build image to see the process.

File: jenkinsfiles/README.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
-- android_jenkinsfile.groovy
-- cordova_jenkinsfile.groovy
+- `android_jenkinsfile.groovy`
+- `cordova_jenkinsfile.groovy`
 
 ## Development
 
-Pipeline files can be modifed for your own needs as long they would launch on the kubernetes node (docker slave).
-Please follow general Jenkins pipelines documentation here: https://jenkins.io/doc/book/pipeline/jenkinsfile/
+Pipeline files can be modified for your own needs as long they would launch on the Kubernetes node (Docker slave).
```
Comment (wtrocki) [inline jenkinsfiles/README.md]: I wrote this but its wrong. We should say: as long as commands/ plugins would lainch..

File: openshift/README.md [hunk 4]
```diff
@@ -7,41 +7,49 @@ Steps
...
+
 To setup simple persistent volume on new cluster execute:
         
-        $ mkdir /tmp/jenkins
-        $ chmod 777 /tmp/jenkins
-        $ oc login -u system:admin
-        $ oc create -f ./sample-pv.json
+        mkdir /tmp/jenkins
+        chmod 777 /tmp/jenkins
+        # creating a cluster wide persistent volume like the one we use requires
```
Comment (wtrocki) [inline openshift/README.md]: Small correction requited

**Comment by wtrocki** [comment]
quick review from phone. Going to check that again in the office

File: openshift/README.md (hunk 4 above)
Comment (aliok) [inline openshift/README.md]: @wtrocki  sorry, this one is not clear to me. what's the required correction?

Comment (wtrocki) [inline openshift/README.md]: Sorry for not being clear. I think that we can just simply say: `Use cluster admin account to create PV.`

**Comment by wtrocki** [comment]
@aliok Great job. Reviewed again! Good to bring back

**Comment by wtrocki** [comment]
@aliok - I'm planning to do some general cleanup and this would made your changes almost impossible to merge. Cherry picking them into my documentation update branch - you can also commit stuff directly there.

**Comment by wtrocki** [comment]
@aliok Changes are now in #5
I had hard time to merge this so best would be to work on #5 to merge it as soon as possible to master.

**Comment by wtrocki** [comment]
Closing as change is included into different PR.

**Pull request closed without merging**